command.go

```diff
@@ -261,6 +268,10 @@ func (c *Command) scanSubcommandHandler(parentg *Group) scanHandler {
 				subc.Aliases = aliases
 			}
 
+			if len(passAfterNonOption) > 0 {
```

Comment:
Does this mean that `foo="no"` is still true?  I realize the code there is all like this so we're not fixing it now, but if this is really the case, this is certainly a red flag.